src/main/java/org/jvnet/hudson/test/JenkinsRule.java

```diff
@@ -510,13 +510,6 @@ public void after() throws Exception {
             try {
                 env.dispose();
             } finally {
-                // Hudson creates ClassLoaders for plugins that hold on to file descriptors of its jar files,
-                // but because there's no explicit dispose method on ClassLoader, they won't get GC-ed until
-                // at some later point, leading to possible file descriptor overflow. So encourage GC now.
-                // see http://bugs.sun.com/view_bug.do?bug_id=4950148
-                // TODO use URLClassLoader.close() in Java 7
```

Comment:
Does Jenkins still work on Java < 7? Maybe it's time to use `URLClassLoader.close()`?

Reply:
Cannot recall offhand if we do already in core. Anyway this is just a super old comment which I am _deleting_.